pgx/src/spi.rs

```diff
@@ -102,6 +109,20 @@ impl TryFrom<libc::c_int> for SpiError {
     }
 }
 
+#[derive(thiserror::Error, Debug)]
+pub enum Error<E: Debug = anyhow::Error> {
```

Comment:
All of the code seems to be within thiserror's power. We do not need to add another dependency.

Reply:
I've removed the dependency on `anyhow`.